example/quantization/imagenet_gen_qsym_mkldnn.py

```diff
@@ -301,7 +301,7 @@ def save_params(fname, arg_params, aux_params, logger=None):
             raise ValueError('unknow calibration mode %s received, only supports `none`, `naive`, and `entropy`'
                              % calib_mode)
         sym_name = '%s-symbol.json' % (prefix + suffix)
-    qsym = qsym.get_backend_symbol('MKLDNN_POST_QUANTIZE')
+    qsym = qsym.get_backend_symbol('MKLDNN_QUANTIZE')
```

Comment:
Do we combine the MKLDNN and MKLDNN_POST_QUANTIZE together?

Reply:
Probably can't. `MKLDNN ` is for fp32 fusion, which will fuse conv+sigmoid together. `MKLDNN_QUANTIZE` is for int8 fusion and quantization, which won't fuse conv+sigmoid because mkldnn can't support this.